rockcraft/oci.py

```diff
@@ -108,13 +121,12 @@ def new_oci_image(
         image_name: str,
         image_dir: Path,
         arch: str,
-        variant: Optional[str] = None,
     ) -> Tuple["Image", str]:
         """Create a new OCI image out of thin air.
 
         :param image_name: The image to initiate, in ``name:tag`` format.
         :param image_dir: The directory to store the local OCI image.
-        :param arch: The architecture of the OCI image to create.
+        :param arch: The architecture of the OCI image to create, in debian format.
```

Comment:
I actually don't know what arch we're supposed to pass to the umoci call in `_config_image()` (line 144); the documentation doesn't specify and apparently (from my testing) it accepts anything

Reply:
did you test with an arch who's goarch is different from the debian arch? like armhf or ppc64le? if not, the easiest way to test might be to either using scalingstack or run rockcraft in `--destructive-mode` from inside a container of that architecture (using qemu)